packages/core/test/interpreter.test.ts

```diff
@@ -950,10 +949,9 @@ Event: {\\"type\\":\\"SOME_EVENT\\"}"
         start: {
           invoke: {
             id: 'child',
-            src: spawnMachine(childMachine),
-            data: {
-              password: 'foo'
-            }
+            src: spawnMachine(() => {
```

Comment:
was the change required? `.withContext` works slightly differently than what was represented by `invoke.data` (not in this case here, but in other cases)

Reply:
Reverted